src/main/java/com/google/ehub/data/EntertainmentItemList.java

```diff
@@ -0,0 +1,27 @@
+package com.google.ehub.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class EntertainmentItemList {
```

Comment:
add javadoc

Reply:
I added javadoc to EntertainmentItemList in last commit.